qz-print/qz-print.nsi.in

```diff
@@ -0,0 +1,50 @@
+!include MUI2.nsh
+
+Name "${manifest.application.name}"
+OutFile "${out.dir}/qz-print-${build.version}-setup.exe"
+
+;-------------------------------
+
+InstallDir "$PROGRAMFILES\QZ"
+
+;-------------------------------
+
+!insertmacro MUI_PAGE_WELCOME
+!insertmacro MUI_PAGE_LICENSE "${basedir}/../LICENSE.txt"
+!insertmacro MUI_PAGE_DIRECTORY
+!insertmacro MUI_PAGE_INSTFILES
+
+!insertmacro MUI_UNPAGE_CONFIRM
+!insertmacro MUI_UNPAGE_INSTFILES
+
+!insertmacro MUI_LANGUAGE "English"
+
+;------------------------------
+
+Section
+
+  SetOutPath "$INSTDIR"
+
+  File /r "${dist.applet.dir}"
```

Comment:
We'll only be installing the socket stuff locally, so the applet directory can be removed here.